examples/LGACSend.ino

```diff
@@ -0,0 +1,258 @@
+  Serial.println("# m : change cooling to air clean, air clean to cooling");
+
+  Serial.println("a=");  // Prompt User for input
+  while (Serial.available() == 0) {  // Wait for user input
+  }
+  char a = Serial.read();  // Read user input into a  
+  switch (a) {
+    case '0':
+    case '1':
+    case '+':
+    case '-':
+    case 'c':
+    case 'h':
+    case 'm':
+      break;
+    default: 
+      Serial.println("b=");  // Prompt User for input
+      while (Serial.available() == 0) {}
+      char b = Serial.read();     	
+  }
+
+  /*
+     # a : mode or temp    b : air_flow, temp, swing, clean, cooling/heating
+     # 18 ~ 30 : temp      0 ~ 2 : flow // on
+     # 0 : off             0
+     # 1 : on              0
+     # 2 : air_swing       0 or 1
+     # 3 : air_clean       0 or 1
+     # 4 : air_flow        0 ~ 3 : flow
+     # + : temp + 1
+     # - : temp - 1
+     # c : cooling         18 ~ 30 temp
+     # h : heating         18 ~ 30 temp
+     # m : change cooling to air clean, air clean to cooling
+  */
+  Serial.print("a : ");
+  Serial.print(a);
+  Serial.print("  b : ");
+  Serial.println(b);
+
+  switch (a) {
+    case '0':  // off
+      ac_power_down();
+      break;
+    case '1':  // on
+      ac_activate(AC_TEMPERATURE, AC_FLOW, AC_HEAT);
+      break;
+    case '2':
+      if (b == '0')
+        ac_change_air_swing(0);
+      else 
+	ac_change_air_swing(1);
+      break;
+    case '3':  // 1  : clean on, power on
+      if (b == '0' | b == '1')
+        ac_air_clean(b);
+      break;
+    case '4':
+      switch (b) {
+        case '1':
+    	  ac_activate(AC_TEMPERATURE, 1, AC_HEAT);
+    	  break;
+        case '2':
+          ac_activate(AC_TEMPERATURE, 2, AC_HEAT);
+          break; 
+        case '3':
+          ac_activate(AC_TEMPERATURE, 3, AC_HEAT);
+          break; 
+        default:
+    	  ac_activate(AC_TEMPERATURE, 0, AC_HEAT);
+      }
+      break;
+    case '+':
+      if (18 <= AC_TEMPERATURE && AC_TEMPERATURE <= 29)
+        ac_activate((AC_TEMPERATURE + 1), AC_FLOW, AC_HEAT);
+      break;
+    case '-':
+      if (19 <= AC_TEMPERATURE && AC_TEMPERATURE <= 30)
+        ac_activate((AC_TEMPERATURE - 1), AC_FLOW, AC_HEAT);
+      break;
+    case 'c':
+    	AC_HEAT = 0;
+    	ac_activate(AC_TEMPERATURE, AC_FLOW, AC_HEAT);
+      break;
+    case 'h':
+    	AC_HEAT = 1;
+    	ac_activate(AC_TEMPERATURE, AC_FLOW, AC_HEAT);
+      break;      
+    case 'm':
+      /*
+          if ac is on,  1) turn off, 2) turn on ac_air_clean(1)
+          if ac is off, 1) turn on,  2) turn off ac_air_clean(0)
+      */
+      if (AC_POWER_ON == 1) {
+        ac_power_down();
+        delay(100);
+        ac_air_clean(1);
+      } 
+      else {
+        if (AC_AIR_ACLEAN == 1) {
+          ac_air_clean(0);
+          delay(100);
+        }
+        ac_activate(AC_TEMPERATURE, AC_FLOW, AC_HEAT);
+      }
+      break;
+    default:
+      if (a >= 18 && a <= 30) {
+        if ( b >= 0 && b <= 2 )
```

Comment:
Ditto for b.
b is `char b = Serial.read();`
It's not going to be around 0-2
You probably mean:
```
if (b >= '0' && b <= '2')
```
And even then it's not what your function is expecting as a value for b.